app/interactors/featured_attachments/update_order_interactor.rb

```diff
@@ -0,0 +1,45 @@
+class FeaturedAttachments::UpdateOrderInteractor < ApplicationInteractor
+  delegate :params,
+           :user,
+           :edition,
+           :attachments,
+           :updater,
+           to: :context
+
+  def call
+    Edition.transaction do
+      find_and_lock_edition
+      reorder_attachments
+      update_edition
+      create_timeline_entry
```

Comment:
I assume we also want to update the preview here? even though it won't yet have an affect for users.

Reply:
So my thinking here was to leave this to be covered by https://trello.com/c/hCrcFkHT/1514-render-featured-attachments-in-the-preview-of-a-publication, which is where it's needed. But it's also conventional to put it in. Updated 👍 .